docs/markdown/Helm/helm-overview.md

```diff
@@ -253,6 +255,20 @@ tests:
 > not affected by the source roots setting. When using `relocated_files`, the files will be relative
 > to the value set in the `dest` field.
 
+### Snapshot testing
+
+Using unit test snapshots is fully supported by Pants as test snapshots are just resources as shown in the previous section.
+
+However, since managing snapshots by hand is quite tedious, Pants gives some utilities to manage them in a simpler way. To automate the update of the snapshots use Pants' `generate-snapshots` goal which we can use to generate the snapshot files:
```

Comment:
```suggestion
Since managing snapshots by hand is quite tedious, Pants provides some utilities to manage them in a simpler way. To generate or update the snapshots, use Pants's `generate-snapshots` goal:
```

Reply:
If the `s` at the end of a word indicates that it's plural, then the possessive is indeed `s'` (like "my friends' names"). But `Pants` is a singular noun ("Pants is a build system", not "Pants are a build system"), so the possessive is `Pants's`. 

If we were talking about trousers, then it is plural ("my Pants are tight", not "my Pants is tight") you would be right, the possessive would be `Pants'`. 

English is a very stupid language...

Reply:
It is indeed! 😂